althea_kernel_interface/src/link_local_tools.rs

```diff
@@ -43,6 +43,23 @@ impl KernelInterface {
         }
     }
 
+    pub fn get_global_device_ip_v4(&self, dev: &str) -> Result<Ipv4Addr, Error> {
+        let output = self.run_command("ip", &["addr", "show", "dev", dev, "scope", "global"])?;
+        trace!("Got {:?} from `ip addr`", output);
+
+        let re = Regex::new(r"inet (\S*?)(/[0-9]+)? scope global").unwrap();
+        let str = String::from_utf8(output.stdout)?;
+        let cap = re.captures(&str);
```

Comment:
Can't the RHS just be on the `if` line below?

Reply:
RHS?